VideoLocker/build.gradle

```diff
@@ -149,6 +150,9 @@ android {
         testInstrumentationRunner 'org.edx.mobile.test.edXInstrumentRunner'
         testHandleProfiling true
         testFunctionalTest true
+
+        // Enabling multidex support.
+        multiDexEnabled true
```

Comment:
@aleffert i remember proguard causes some issues for one or two libraries we used?
i can do some research and create a separate PR for it.